flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/ProjectionTranslator.scala

```diff
@@ -221,6 +221,23 @@ object ProjectionTranslator {
     projectList
   }
 
+  def translateOverWindows(
+      exprs: Seq[Expression],
+      overWindows: Array[OverWindow]): Seq[Expression] = {
```

Comment:
Add `tableEnv` parameter to identify whether this is a batch or a streaming query. 
In case of a streaming query, check that `overwindow.orderBy` is an `UnresolvedFieldReference` and translates `rowtime` to a `RowTime` expression and `proctime` to a `ProcTime` expression.